src/commands/server/status.ts

```diff
@@ -0,0 +1,74 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { Command, flags } from '@oclif/command'
+import { cli } from 'cli-ux'
+import * as notifier from 'node-notifier'
+
+import { KubeHelper } from '../../api/kube'
+import { VersionHelper } from '../../api/version'
+import { cheNamespace } from '../../common-flags'
+
+export default class List extends Command {
+  // Implementation-Version it is a property from Manifest.ml inside of che server pod which indicate Eclipse Che build version.
+  readonly chePreffixVersion = 'Implementation-Version: '
+  readonly cheServerSelector = 'app=che,component=che'
+
+  static description = 'status Eclipse Che server'
+  openshiftOauth = false
+  cheVersion = 'UNKNOWN'
+  cheUrl = 'UNKNOWN'
+
+  kube = new KubeHelper(flags)
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    chenamespace: cheNamespace,
+  }
+
+  async run() {
+    const { flags } = this.parse(List)
+    const cr = await this.kube.getCheCluster(flags.chenamespace)
+
+    if (cr && cr.spec && cr.spec.auth && typeof cr.spec.auth.openShiftoAuth === 'boolean') {
+      this.openshiftOauth = true
+    }
+
+    if (cr && cr.status && cr.status.cheURL) {
+      this.cheUrl = cr.status.cheURL
```

Comment:
The problem is that CR might not exists if `helm` was used.
So, grab the cheURL from the `che` configmap

Reply:
@tolusha I've reused https://github.com/che-incubator/chectl/blob/master/src/api/che.ts#L102 instead of get url from configMap